docker/nginx/nginx.conf

```diff
@@ -0,0 +1,82 @@
+worker_processes 4;
+
+events { worker_connections 1024; }
+
+http {
+	upstream app_server {
+        	server app-1:3000 fail_timeout=5s max_fails=5;
+	}
+
+	server {
+	    listen 80 default_server;
+	    listen [::]:80 default_server;
+	
+	    charset utf-8;
+	
+	    access_log /var/log/nginx/openhab.org-access.log;
+	    error_log /var/log/nginx/openhab.org-error.log;
+	    client_max_body_size 300m;
+	
+	    root /opt/openhabcloud;
+	
+	    location ~ /.well-known {
+	      allow all;
+	    }
+	
+		location /css {
```

Comment:
@FlorianSW your proposal with the include /etc/nginx/mime.types; didn't work.
I will try other possibilities...Style is still missing.